src/h5web/toolbar/controls/DomainSlider/ScaledSlider.tsx

```diff
@@ -0,0 +1,68 @@
+import type { ReactElement, Ref } from 'react';
+import { FiSkipBack, FiSkipForward } from 'react-icons/fi';
+import ReactSlider from 'react-slider';
+import { Domain, extendDomain, ScaleType } from '../../../../packages/lib';
+import { createAxisScale } from '../../../vis-packs/core/utils';
+import styles from './DomainSlider.module.css';
+import Thumb from './Thumb';
+
+const SLIDER_RANGE: Domain = [1, 100];
+
+interface Props {
+  rawDomain: Domain;
+  dataDomain: Domain;
+  disabled?: boolean;
+  isAutoMin: boolean;
+  isAutoMax: boolean;
+  onChange: (domain: Domain) => void;
+  onAfterChange: (domain: Domain) => void;
+}
+
+const EXTEND_FACTOR = 0.2;
```

Comment:
This should go above the `interface Props` with `SLIDER_RANGE` or both should go under `interface Props`